lte/protos/subscriberdb.proto

```diff
@@ -271,3 +273,22 @@ message ListSubscribersResponse {
   // Deprecated
   reserved 3, 4;
 }
+
+message SuciProfile {
```

Comment:
Can you add comments to link to the spec that's being followed for this?

Reply:
done